Resources/Prototypes/_CD/Entities/Mobs/Customization/Markings/ipc_front.yml

```diff
@@ -0,0 +1,351 @@
+#region auto generated
```

Comment:
If this was automatically generated, could you provide the script?

Reply:
the script is messy, has hardcoded states and was pretty much just so that I didn't need to write 30 or so different yml entries

i don't believe its that necessary to supply the script (and the fact that i deleted it on my disk)

Reply:
Alright, then can you remove the comment so people are not afraid to modify the file?

Reply:
sure